### PR TITLE
switch over to the ClientPipelined model, and batch insert blocks

### PR DESCRIPTION
**Pull request by cleverca22**

### CODE REVIEW

File: cardano-explorer-node/src/Explorer/Node.hs [hunk 1]
```diff
@@ -308,13 +323,40 @@ localInitiatorNetworkApplication Proxy trce pInfoConfig =
...
             liftIO $ logDbState trce
-            runPeer
+            actionQueue <- newSingleTypeQueueIO
+            (metrics, server) <- runRegistryT $ do
+              metrics <- makeMetrics
+              registry <- RegistryT ask
+              server <- liftIO $ async $ runReaderT (unRegistryT $ serveHttpTextMetricsT 8080 []) registry
+              pure (metrics, server)
+            dbThread <- async $ startDbThread trce actionQueue metrics
+            wut <- runPipelinedPeer
```
Comment (cleverca22) [inline cardano-explorer-node/src/Explorer/Node.hs:335]: what is the proper way to return `Void`? the constructor doesnt seem to be exported

**Review by erikd — DISMISSED** [review]

File: cardano-explorer-node/src/Explorer/Node.hs [hunk 2]
```diff
@@ -392,6 +447,54 @@ localTxSubmissionCodec
...
+    if (newSize == 0)
+      then writeTVar (stqContains actionQueue) Empty
+      else pure ()
+    pure (eActs, oldSize, newSize)
+  Gauge.set (fromIntegral $ oldSize) mQueuePre
+  Gauge.set (fromIntegral $ newSize) mQueuePost
+  case eActs of
+    Right actions -> do
+      let
+        youSawNothing (ApplyBlock blk) = blk
```
Comment (cleverca22) [inline cardano-explorer-node/src/Explorer/Node.hs:483]: a better option, is maybe to have 2 queues, each of a single type, so i dont have to deal with the sum-type here, and then use the `OnlyContains` type to control which queue i read from

File: cardano-explorer-node/src/Explorer/Node.hs (hunk 1 above)
Comment (erikd) [reply cardano-explorer-node/src/Explorer/Node.hs:335]: Ah missed that.

Two best way to do that are:
```
    void runPipelinedPeer
```
or
```
   _ <- runPipelinedPeer
```

**Comment by erikd** [comment]
Superceded by https://github.com/input-output-hk/cardano-explorer/pull/77

**Pull request closed without merging**